sbin/build.sh

```diff
@@ -224,9 +230,9 @@ runTheOpenJDKConfigureCommandAndUseThePrebuildConfigParams()
 buildOpenJDK()
 {
   if [[ "$OS_MACHINE" == "s390x" ]] || [[ "$OS_MACHINE" == "armv7l" ]] ; then
```

Comment:
We can still make it buildable from CLI, as I had suggested to provide a script i.e make-from-local-machine.sh that called makejdk with the different ENV variables.